<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQchangedetail</class>
 <widget class="QWidget" name="QQchangedetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>560</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>560</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>560</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Noto Sans Mono CJK SC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>账号:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="idLab">
        <property name="font">
         <font>
          <family>Noto Sans Mono CJK SC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLable</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Noto Sans Mono CJK SC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>注册:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="regLab">
        <property name="font">
         <font>
          <family>Noto Sans Mono CJK SC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLable</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Noto Sans Mono CJK SC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>昵称:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLdt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Noto Sans Mono CJK SC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>生日:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumTime">
         <time>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Noto Sans Mono CJK SC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>住址:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LocaLdt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="confrimBtm">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>确定修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitBtm">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>返回信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
